### PR TITLE
[validation] Add ShortPhpConfigRule

### PR DESCRIPTION
**Pull request by samsonasik**
Currently, there short maximum lines = 100 is on php contents, but on php config, it doesn't have this validation.

This PR add `ShortPhpConfigRule` for config rule, which the config validation config box, which shared with "Create custom rule" page to avoid too many lines added on the page.

This `ShortPhpConfigRule` have 200 lines maximum.

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: src/Validation/Rules/ShortPhpLinesTrait.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+namespace App\Validation\Rules;
+
+use Closure;
+use Illuminate\Translation\PotentiallyTranslatedString;
+
+/**
+ * @internal
+ * @method int getInputLineLimit()
+ */
+trait ShortPhpLinesTrait
```
Comment (TomasVotruba) [inline src/Validation/Rules/ShortPhpLinesTrait.php:14]: Instead of trait with this `@method` magic mess, use normal service.

Comment (samsonasik) [reply src/Validation/Rules/ShortPhpLinesTrait.php]: sure, updated to use normal service https://github.com/rectorphp/getrector-com/pull/2490/commits/618cbd13acd50c252ecf8cf2d969621d1d9b4675

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Review by TomasVotruba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thanks 🙏👍

**Pull request merged**